tests/layer_tests/tensorflow_tests/test_tf_UniqueWithCounts.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestUnique(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
```

Comment:
You don't need to override `_prepare_input` here. CommonLayerTest class already has this method, which can be used in most cases: https://github.com/openvinotoolkit/openvino/blob/e2e9d6056f066e77722265e17c81a7071e0804c7/tests/layer_tests/common/layer_test_class.py#L149

We need to ovverride _prepare_input only for complex type.